include/index.h

```diff
@@ -325,8 +325,10 @@ template <typename T, typename TagT = uint32_t, typename LabelT = uint32_t> clas
     // Graph related data structures
     std::vector<std::vector<uint32_t>> _final_graph;
 
+    T *_data = nullptr; // coordinates of all base points
     // Dimensions
     size_t _dim = 0;
+    size_t _aligned_dim;
```

Comment:
We prefer not to have aligned dim in the index class and instead ask the datastore for it